mne_bids/write.py

```diff
@@ -46,6 +46,17 @@ def _is_numeric(n):
     return isinstance(n, (np.integer, np.floating, int, float))
 
 
+def _stamp_to_dt(utc_stamp):
+    """Convert timestamp to datetime object in Windows-friendly way."""
+    # The min on windows is 86400,
+    # fsee https://github.com/home-assistant/appdaemon/issues/83
+    stamp = [int(s) for s in utc_stamp]
+    if len(stamp) == 1:  # In case there is no microseconds information
+        stamp.append(0)
+    return (datetime.fromtimestamp(0, tz=timezone.utc) +
```

Comment:
If 86400 is the minimum, then how does 0 work? 

side note, don't squash commits before the PR is approved to merge. It prevents me from looking at older failed builds (e.g., I can't find the appveyor build that failed anymore) and slows down the review process.